Packs/Zscaler/.pack-ignore

```diff
@@ -1,2 +1,2 @@
 [file:README.md]
-ignore=RM106
\ No newline at end of file
+ignore=RM104
```

Comment:
You wanted to add or remove the existing?

Reply:
Didn't get what you mean